deploy_to_live.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Testing comments.
Also, this seems to only push from test to live. Rename script "deploy_test_to_live.sh"?